### DISCUSSION TITLE
Recently, the CI is failing due to docs building failures

### DISCUSSION THREAD

**Discussion by svlandeg** [fastapi/fastapi#12011] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Description

Recently, PRs have started failing the CI due to failing some of the document building checks. It looks like a regression may have been introduced on `master` causing this.

Example PRs that failed today:
 * https://github.com/fastapi/fastapi/pull/11876
 * https://github.com/fastapi/fastapi/pull/11515

The error is caused by the following error:
```
No such file or directory: '/home/runner/work/fastapi/fastapi/docs/de/docs/img/icon-white.svg'
```

And the same thing in other languages:
```
No such file or directory: '/home/runner/work/fastapi/fastapi/docs/bn/docs/img/icon-white.svg'
```

**Comment by svlandeg** [comment]
Then again, this PR did just succeed on the CI 🤔 

-  https://github.com/fastapi/fastapi/pull/11996

**Comment by svlandeg** [comment]
One potential difference I can find is that the failing PRs don't seem to have access to the `INSIDERS_FILE` env var:

```
Insiders file None
Building docs for: fr
```

While the more recent, passing PR does:
```
Insiders file ../en/mkdocs.insiders.yml
Using insiders
Building docs for: fr
```

All this originates from the code
```
insiders_env_file = os.environ.get("INSIDERS_FILE")
print(f"Insiders file {insiders_env_file}")
```

and the env var is set by 
```
    if is_mkdocs_insiders():
        os.environ["INSIDERS_FILE"] = "../en/mkdocs.insiders.yml"
```

**Reply by tiangolo** [reply, marked as answer]
Thank you! This was handled here: https://github.com/fastapi/fastapi/pull/12016
